### ISSUE TITLE
Stops without warning when playing random songs

### ISSUE THREAD

**Issue by jelmerdereus** [osteenbergen/slackify#19]
Happened on Thursday and Friday. No idea why. It does continue when hitting `@slackify: n` .

**Comment by osteenbergen** [comment]
Found multiple issues in the logs: http://pastebin.com/fDWbFnNn

Most of them relate to posting messages back to the channel. If this happens at the end of a song it could happen that the the new song is never started. 
Will catch these exceptions during play_end to ensure that a new song will start

**Comment by osteenbergen** [comment]
New error, same issue:
```
streaming error: <ErrorType.OTHER_PERMANENT: 10>
```

Song: Mastodon - Colony Of Birchmen
URI: ```spotify:track:6nmsnhkJYvyYu8TheUa0Xf```

**Comment by Wqrld** [comment]
im having the same problem, its something with pyspotify but idk what

**Comment by osteenbergen** [comment]
If you see error messages it would help, but i know of three reasons why it
could happen

1. You are sharing your spotify account and someone else presses play on
their phone
2. You have a spotty internet connection
3. The song slackify starts to play is region restricted and the playback
halts


The first and thrid reasons was why Jelmer opened thus bug report. I
haven't looked at solving those with nice error messages. The second
usually results in clear errors in the terminal running python.

Please share more details so I can help you.

**Comment by Wqrld** [comment]
i had something that hijacked my dns and after switching my dns it worked for me. And i just found out this was one extreme necro 

tldr; issue fixed for me

en ik zag net dat jullie beiden nl waren dus cool project :)
